test/data/rules/wildcard

```diff
@@ -0,0 +1,32 @@
+! model		=	keycodes
+  *		=	evdev
+
+! model		=	geometry
+  *		=       pc(pc104)
+
+! layout	variant	=	symbols
+  *		*	=	pc+%l%(v)
+
+! layout[1]	variant[1]	=	symbols
+  *		*		=	pc+%l[1]%(v[1])
+
+! layout[2]	variant[2]	=	symbols
+  *		*		=	+%l[2]%(v[2]):2
+
+! layout[3]	variant[3]	=	symbols
+  *		*		=	+%l[3]%(v[3]):3
+
+! layout[4]	variant[3]	=	symbols
+  *		*		=	+%l[4]%(v[4]):4
+
+! model		layout		=	compat
+  *		*		=	complete
+
+! model		layout[1]	=	compat
+  *		*		=	complete
+
+! model		layout[2]	=	compat
+  *		*		=	complete
+
+! model		=	types
+  *		=	complete
```

Comment:
might be github's diff view but it looks like the indentation/alignment here is quite off which makes this file a bit hard to look at.

Reply:
You need to set tab size to 8, as in `xkeyboard-config`. I changed it to spaces to avoid this issue.

Reply:
yeah, it rendered fine in the hunk above but this one looked weird. tab size is (as it should be!) 8 by default anyway, I didn't even know that was a setup in github :)

Anyway, all good, thanks!